jerry-debugger/jerry-client-ws.py

```diff
@@ -729,6 +737,46 @@ def parse_source(debugger, data):
         logging.debug("No pending breakpoints")
 
 
+def src_check_args(debugger, is_src_cmd, args):
+    if args:
+        try:
+            line_num = int(args)
+            if line_num < 0:
+                print("Error: Non-negative integer number expected")
+                return -1
+
+            return line_num
+        except ValueError as val_errno:
+            print("Error: Non-negative integer number expected: %s" % (val_errno))
+            return -1
+    elif is_src_cmd:
+        return debugger.default_viewrange
+    else:
+        print("Display should be called with an argument")
+        return -1
+
+
+def print_source(debugger, line_num):
+    last_bp = debugger.last_breakpoint_hit
+    if last_bp:
```

Comment:
One of my comment is missing:

    if not last_bp:
      return

And remove the indentation form the code after.